README.md

```diff
@@ -57,22 +57,34 @@ You can use docker-compose, to start a database. To do that:
 }
 ```
 
-3. Copy .env.example to .env, and update the DATABASE_URL environment variable
+3. Create an .env.development file in the root "/" directory and copy the contents of .env.example. Then update the DATABASE_URL environment variable
 
 ```
-DATABASE_URL=postgresql://wallet_user:secret@localhost:5432/wallet_user?ssl=no-verify
+DATABASE_URL=postgresql://wallet_user:secret@localhost:5432/wallet_user
```

Comment:
Can you make it `?false` or put it as an option?  Also, can you add a disclaimer that ssl=no-verify should be used if the database is not locally hosted